freqtrade/freqai/RL/Base4ActionRLEnv.py

```diff
@@ -0,0 +1,134 @@
+import logging
+from enum import Enum
+
+from gym import spaces
+
+from freqtrade.freqai.RL.BaseEnvironment import BaseEnvironment, Positions
+
+
+logger = logging.getLogger(__name__)
+
+
+class Actions(Enum):
+    Neutral = 0
+    Exit = 1
+    Long_enter = 2
+    Short_enter = 3
+
+
+class Base4ActionRLEnv(BaseEnvironment):
```

Comment:
It is another way, in Reinforcement learning, if you use one exit instead of two, the neural net gives another weights to action, it is a good example how to modify the the whole env. Do you want us to make an example "Reinforcement Learner" file as an example?

Reply:
> example "Reinforcement Learner"

You mean a strategy file?

Reply:
Base4 would be used if the user wants the agent to look at all exits the same way. Base5 would be used if the user wants the agent to learn about exiting shorts differently from exiting longs. 

We can add this to the docs. We had example strats around but you've in the past told us to remove example strats from the source. Therefore, example strats were removed from the source.

Reply:
I have added a section in the docs to explain how to select the environment, and how to customize the environment.